src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -555,6 +555,9 @@ private static string GetNextArgument(string arguments, ref int i)
                 {
                     if (inQuotes && i < arguments.Length - 1 && arguments[i + 1] == '"')
                     {
+                        // Two consecutive double quotes inside an inQuotes region should result in a literal double quote 
+                        // (the parser is left in the inQuotes region).
+                        // This behavior is not part of the spec linked above, but is compatible with CRT and .NET Framework.
```

Comment:
spec linked above where?

Reply:
https://github.com/dotnet/corefx/pull/25955/files#diff-015aad2514de8320cbfe19ed2b8ee9a7R493

Not sure if repeating the link is useful ...

Reply:
Yeah, that link is on a different method, it's confusing what you are referring to, maybe mentioning above the other method makes sense.